tests/testthat/test-vivax-eq.R

```diff
@@ -0,0 +1,86 @@
+test_that('Initial states are consistent with equilibrium', {
+  skip_on_ci()
```

Comment:
We could run this on the CI? It seems valuable and quick to run. If we're not doing this for falciparum, maybe we should be.